BetterPasswordHandler.inc.php

```diff
@@ -42,4 +42,48 @@ function signIn($args, $request) {
 		));
 		$templateMgr->display('frontend/pages/userLogin.tpl');
 	}
+        
+        /**
+        * Store the uploaded blacklists files
+        * @return boolean JSONMessage
+        */
+        function uploadBlacklists($args, $request) {
+            import('lib.pkp.classes.file.PrivateFileManager');
+            $privateFileManager = new PrivateFileManager();
+            
+            $uploadedFile = $_FILES['uploadedFile'];
+            $destFilePath = $privateFileManager->getBasePath() . DIRECTORY_SEPARATOR . 'betterPassword' . DIRECTORY_SEPARATOR . 'blacklists' . DIRECTORY_SEPARATOR . $uploadedFile['name'];
```

Comment:
Doesn't the `$_FILES['uploadedFile']['name']` value come directly from the end user?  How can we trust this value?  What if a malicious user submitted "../../../../../../../etc/passwd"?